README.rst

```diff
@@ -18,9 +26,104 @@ There is no released code API.
   Sometimes, it may not work at all.
 
 Each official release is tagged in the repository. The `change
-history`_ describes every version.
+history` _describes every version.
+
+ROS usage
+---------
```

Comment:
There's [already a doc job for axis_camera](https://github.com/ros/rosdistro/blob/7933ffbb9c826a27a56be7965f16a2415b92d484/indigo/distribution.yaml#L469), so it'll happen by itself if everything is set up correctly in the repo.

You can test it locally by running rosdoc_lite yourself in your workspace: http://wiki.ros.org/rosdoc_lite